src/languageSupport/languages/flux/lsp/connection.ts

```diff
@@ -186,33 +181,76 @@ class LspConnectionManager {
     })
   }
 
+  _alignInvisibleDivToEditorBlock() {
+    // elements in monaco-editor. positioned by editor.
```

Comment:
this method is code previously in `_setEditorSyncToggle()`, now in a helper method which can be used in the callback `_setEditorBlockStyle()`.